jaxrs-tck/src/main/resources/jakarta/ws/rs/tck/signaturetest/sig-test.map

```diff
@@ -0,0 +1,20 @@
+#
+# Copyright (c) 2018, 2020 Oracle and/or its affiliates. All rights reserved.
+#
+# This program and the accompanying materials are made available under the
+# terms of the Eclipse Public License v. 2.0, which is available at
+# http://www.eclipse.org/legal/epl-2.0.
+#
+# This Source Code may also be made available under the following Secondary
+# Licenses when the conditions for such availability set forth in the
+# Eclipse Public License v. 2.0 are satisfied: GNU General Public License,
+# version 2 with the GNU Classpath Exception, which is available at
+# https://www.gnu.org/software/classpath/license.html.
+#
+# SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+#
+
+###############################################################
+# The signature test mapping file for the JAX-RS TCK.
+###############################################################
+jakarta.ws.rs=3.1-SNAPSHOT
```

Comment:
This likely should be called 3.1. I assume the name does not need to correspond to the API version.

Reply:
Renamed to 3.1.0. I think we can refactor the code itself to not require the mapping file if we are to keep only one signature file for a release. But the same is used as to comply with the current signature test run framework available in jakartaee-tck. The version here is used to pick the right signature file(corresponding to the api version) to be used to run the test. The file is named as jakarta.ws.rs.sig_3.1.0.